### ISSUE TITLE
TCP and UDP tables set to null crash osquery

### ISSUE THREAD

**Issue by Breakwell** [osquery/osquery#7188]
# Bug report

### What operating system and version are you using?

```
 version = 10.0.19042
   build = 19042
platform = windows
```

### What version of osquery are you using?

```
version = 4.8.0
```

### What steps did you take to reproduce the issue?

It seems when we construct the [WinSockets](https://github.com/osquery/osquery/blob/master/osquery/tables/networking/windows/process_open_sockets.cpp#L43) object in [process_open_sockets](https://github.com/osquery/osquery/blob/master/osquery/tables/networking/windows/process_open_sockets.cpp) it can error and not set the [table objects](https://github.com/osquery/osquery/blob/master/osquery/tables/networking/windows/win_sockets.h#L43). This then means when we call [parseSocketTable](https://github.com/osquery/osquery/blob/master/osquery/tables/networking/windows/process_open_sockets.cpp#L95) we can potentially hit a null and crash.

### What did you expect to see?

When we fail to set the tables and use the process_open_sockets table  osquery should log a warning or retry to create the tables.

### What did you see instead?

Osquery crashed with a  null pointer error.

**Comment by AndreaMarangoni** [comment]
This is fixed by [#8419](https://github.com/osquery/osquery/pull/8419) and can  be closed

**Comment by directionless** [comment]
Fixed by #8419
